relay-metrics/src/bucket.rs

```diff
@@ -538,7 +537,7 @@ pub struct Bucket {
     ///
     /// Namespaces and units must consist of ASCII characters and match the regular expression
     /// `/\w+/`. The name component of MRIs consist of unicode characters and must match the
-    /// regular expression `/\w[\w\d_-.]+/`. Note that the name must begin with a letter.
+    /// regular expression `/\w[\w\-.]*/`. Note that the name must begin with a letter.
```

Comment:
Should the `.` be escaped? We mean literal dot, not any character, right?

Reply:
Inside of character groups, dot is literal.